src/app/views/discover/home/home.ts

```diff
@@ -33,19 +28,45 @@ import AppDiscoverHomeCommunities from './_communities/communities.vue';
 	resolver: () => Api.sendRequest('/web/discover'),
 })
 export default class RouteDiscoverHome extends BaseRouteComponent {
-	@State app!: Store['app'];
+	@AppState
+	user!: Store['app'];
+
+	@AppState
+	userBootstrapped!: AppStore['userBootstrapped'];
 
 	featuredItem: FeaturedItem | null = null;
 	featuredCommunities: Community[] = [];
 	featuredFireside: Fireside | null = null;
+	heroPosts: FiresidePost[] = [];
+
+	split: null | 'default' | 'hero' = null;
 
 	routeCreated() {
 		Meta.setTitle(null);
+
+		this.$watch(
+			() => this.userBootstrapped,
+			userBootstrapped => {
+				if (!userBootstrapped) {
+					return;
+				}
+
+				// If we bootstrapped and are logged out, then we've decided to
+				// show this as a split test. If we're logged in, we always use
+				// the default, since it's used as the discover page.
+				if (!this.user) {
+					trackExperimentEngagement(configGuestHome);
+					this.split = configGuestHome.value;
+					this.split = 'hero';
```

Comment:
This second line should be removed, yeah?

Reply:
oh dang, yeah, haha